src/lib/tabwidget/tabicon.cpp

```diff
@@ -113,4 +128,26 @@ void TabIcon::paintEvent(QPaintEvent* event)
         p.drawPixmap(r, m_animationPixmap, QRect(m_currentFrame * pixmapSize, 0, pixmapSize, pixmapSize));
     else
         p.drawPixmap(r, m_sitePixmap);
+
+#if QT_VERSION >= QT_VERSION_CHECK(5,7,0)
+    if (m_audioIconDisplayed) {
+        if (m_tab->isMuted())
+            p.drawPixmap(QPointF(width() * 0.25, 0), m_audioMutedPixmap);
+        else
+            p.drawPixmap(QPointF(width() * 0.25, 0), m_audioPlayingPixmap);
+    }
+#endif
+}
+
+#if QT_VERSION >= QT_VERSION_CHECK(5,7,0)
+void TabIcon::mousePressEvent(QMouseEvent *event)
+{
+    qreal x = event->localPos().x();
```

Comment:
I don't get this comment.
This method should only override if version is >= 5.7.0